medusa/medusacli.py

```diff
@@ -17,6 +17,8 @@
 from gevent import monkey
 
 import medusa.utils
+from medusa import backup_node
+from medusa.backup_manager import BackupMan
```

Comment:
These two line should be placed after `monkey.patch_all()` otherwise it'll break some stuff in the orchestration of backups.
Also `from medusa import backup_node` should replace `import medusa.backup_node` on line 40 which isn't necessary anymore.